server/sonar-server/src/test/java/org/sonar/server/organization/ws/CreateActionTest.java

```diff
@@ -109,14 +116,41 @@ public void verify_response_example() throws URISyntaxException, IOException {
   }
 
   @Test
-  public void request_fails_if_user_does_not_have_SYSTEM_ADMIN_permission() {
+  public void request_fails_if_user_does_not_have_SYSTEM_ADMIN_permission_and_logged_in_user_can_not_create_organizations() {
     expectedException.expect(ForbiddenException.class);
     expectedException.expectMessage("Insufficient privileges");
 
     executeRequest("name");
   }
 
   @Test
+  public void request_succeeds_if_user_has_SYSTEM_ADMIN_permission_and_logged_in_user_can_not_create_organizations() {
+    giveUserSystemAdminPermission();
+    mockForSuccessfulInsert(SOME_UUID, SOME_DATE);
```

Comment:
I wonder...

your suggestion is not a 100% replacement. I won't set now in field initialization but only in `mockForSuccessfulInsert`. This way, any unexpected call will fail and that's good :)
It will also keep clear in test method that the date of the organization is the one provided at the beginning of the test.